packages/blade/src/components/Amount/Amount.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title } from '@storybook/addon-docs';
+import type { AmountProps } from './Amount';
+import { Amount as AmountComponent } from './Amount';
+import BaseBox from '~components/Box/BaseBox';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+import { Text } from '~components/Typography';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+
+const Page = (): React.ReactElement => {
```

Comment:
Can you add a story where Amount is used inline with Title, Heading and Text components?